web/src/settings_invites.ts

```diff
@@ -316,6 +319,15 @@ export function on_load_success(
 
         $(".dialog_submit_button").attr("data-invite-id", invite_id);
     });
+
+    $("#admin-invites-list").on("click", ".clear_filter", (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+
+        const $container = $("#admin-invites-list").find(".search_container");
+        const $filter = $container.find(".search");
+        invite_list_widget.clear_text_filter($filter);
+    });
```

Comment:
Meanwhile, could this potentially by implemented as an option to `ListWidget` rather than code we copy into each list widget with a filter component? `filter` is already a parameter to `ListWidget`, so it's intended to have some native support for filtering.

Reply:
If I am not mistaken then you are suggesting not to use that filter as a parameter in clear_text_filter right?
I have updated my PR to do so.
Kindly check my current approach.
Thanks :)